File: app/src/main/java/com/dlovid/short/MainActivity.java
package com.dlovid.short;
import android.app.Activity;
import android.os.Bundle;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;

public class MainActivity extends Activity {
    private WebView webView;
    // GANTI URL INI DENGAN URL DLOVID KAMU YANG ASLI
    private static final String DLOVID_URL = "https://dlovid.com";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        webView = new WebView(this);
        setContentView(webView);
        WebSettings ws = webView.getSettings();
        ws.setJavaScriptEnabled(true);
        ws.setDomStorageEnabled(true);
        ws.setAllowFileAccess(true);
        webView.setWebViewClient(new WebViewClient());
        webView.loadUrl(DLOVID_URL);
    }
    @Override
    public void onBackPressed() {
        if (webView.canGoBack()) webView.goBack();
        else super.onBackPressed();
    }
}

File: app/src/main/java/com/dlovid/short/MainActivity.kt
package com.dlovid.short

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.midtrans.sdk.SdkUIFlowBuilder

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // INIT MIDTRANS QRIS 15K
        SdkUIFlowBuilder.init()
            .setClientKey(MidtransConfig.CLIENT_KEY)
            .setContext(this)
            .setMerchantBaseUrl("https://app.sandbox.midtrans.com/snap/v1/")
            .enableLog(true)
            .buildSDK()

        // Tombol ke Bonus VIP
        try {
            val btnBonus = findViewById<Button>(R.id.btnBonus)
            btnBonus.setOnClickListener {
                startActivity(Intent(this, BonusActivity::class.java))
            }
        } catch (e: Exception) {
        }
    }
}
